eml/services/utilityFunctions.js

```diff
@@ -252,22 +251,27 @@ export function findIndexOfUncompletedComp(student, courseId, sectionId) {
 }
 
 export async function handleLastComponent(comp, course, navigation) {
-  const student = await StorageService.getStudentInfo();
-  const isComplete = isSectionCompleted(student, comp.parentSection);
+
+  // For future reference
+  // const student = await StorageService.getStudentInfo();
+  // const isComplete = isSectionCompleted(student, comp.parentSection);
   
-  if (isComplete) { 
-    navigation.reset({
-      index: 0,
-      routes: [
-        {
-          name: 'CompleteSection',
-          params: { 
-            parsedCourse: course, 
-            sectionId: comp.parentSection }
-        },
-      ],
-    });
-  } else {
-    console.log('Section not complete');
-  }
+  navigation.reset({
+    index: 0,
+    routes: [
+      {
+        name: 'CompleteSection',
+        params: { 
+          parsedCourse: course, 
+          sectionId: comp.parentSection }
+      },
+    ],
+  });
+
+  // For future reference
+  // if (isComplete) { 
+  // Code above with naviagtion
+  // } else {
+  //   console.log('Section not complete - navigate to retry exercises TBD');
+  // }
```

Comment:
same, what does this mean/how can it be used?

Reply:
see other answer ≽ܫ≼